src/registrar/templates/includes/input_with_errors.html

```diff
@@ -52,9 +52,14 @@
   {% if field.errors %}
     <div id="{{ widget.attrs.id }}__error-message">
     {% for error in field.errors %}
-      <span class="usa-error-message" role="alert">
-        {{ error }}
-      </span>
+      <div class="usa-error-message display-flex" role="alert">
+        <div>
```

Comment:
Is the extra div wrapper needed here?

Reply:
Nope.  Remnant from former cleanup.  Good catch